content/en/docs/tasks/observability/distributed-tracing/sampling/index.md

```diff
@@ -0,0 +1,235 @@
+such as [Probability Sampling](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.31.0/specification/trace/tracestate-probability-sampling.md).
+
+Such samplers can then be paired with the the [`OpenTelemetryTracingProvider`](/docs/reference/config/istio.mesh.v1alpha1/#MeshConfig-ExtensionProvider-OpenTelemetryTracingProvider).
+
+{{< quote >}}
+The sampler implementation resides in the proxy and can be found in
+[Envoy OpenTelemetry Samplers](https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/trace/opentelemetry/samplers#opentelemetry-samplers).
+{{< /quote >}}
+
+Current custom sampler configurations in Istio:
+
+- [Dynatrace Sampler](/docs/reference/config/istio.mesh.v1alpha1/#MeshConfig-ExtensionProvider-OpenTelemetryTracingProvider-DynatraceSampler)
+
+Custom samplers are configured via `Meshconfig`. Here is an example of configuring the Dynatrace sampler:
+
+{{< text syntax=yaml snip_id=none >}}
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+spec:
+  meshConfig:
+    extensionProviders:
+    - name: otel-tracing
+      opentelemetry:
+        port: 443
+        service: abc.live.dynatrace.com/api/v2/otlp
+        http:
+          path: "/api/v2/otlp/v1/traces"
+          timeout: 10s
+          headers:
+            - name: "Authorization"
+              value: "Api-Token dt0c01."
+        dynatrace_sampler:
+          tenant: "abc"
+          cluster_id: 123
+{{< /text >}}
+
+## Order of precedence
+
+With multiple ways of configuring sampling, it is important to understand
+the order of precedence of each method.
+
+When using the random percentage sampler the order of precedence is:
+
+`Telemetry API` > `Pod Annotation` > `MeshConfig`.
+
+That means, if a value is defined in all of the above, the value on the `Telemetry API` is the one selected.
+
+When a custom OpenTelemetry sampler is configured, the order of precedence is:
+
+`Custom OTel Sampler` > (`Telemetry API` | `Pod Annotation` | `MeshConfig`)
+
+That means, if a custom OpenTelemetry sampler is configured, it overrides all the others methods.
+Additionally, the random percentage value is set to `100` and cannot be changed. This is important
+because the custom sampler needs to receive 100% of spans to be able to properly perform its decision.
+
+## Deploy the Bookinfo Application
+
+Deploy the [Bookinfo](/docs/examples/bookinfo/#deploying-the-application) sample application.
+
+## Generating traces using the Bookinfo sample
+
+1.  When the Bookinfo application is up and running, access `http://$GATEWAY_URL/productpage`
+    one or more times to generate trace information.
+
+    {{< boilerplate trace-generation >}}
+
+## Cleanup
+
+1.  Remove the Telemetry resource:
+
+    {{< text syntax=bash snip_id=cleanup_telemetry >}}
+    $ kubectl delete telemetry otel-demo
+    {{< /text >}}
+
+1.  Remove any `istioctl` processes that may still be running using control-C or:
+
+    {{< text syntax=bash snip_id=none >}}
+    $ killall istioctl
```

Comment:
the test is doing istioctl uninstall , as that is how you did the installation.. i think better to do so, to align with all other documents?

Reply:
I changed it now to use the same as in the test with `uninstall`. Hope the test passes.